pytorch_toolkit/formula_recognition/README.md

```diff
@@ -81,6 +81,36 @@ In addition to common parameters you can specify the following arguments:
 - `train_transforms_list` - similiar to `val_transforms_list`
 - `epochs` - number of epochs to train
 
+One can use some pretrained models. Right now two models are available:
+* medium model:
+    * [checkpoint link](https://download.01.org/opencv/openvino_training_extensions/models/formula_recognition/medium_photograped_0185.pth)
+    * digits, letters, some greek letters, fractions, trigonometric operations are supported; for more details, please, look at corresponding vocab file
+    * to use this model, please, change these fields in the config file:
+    ```
+    backbone_config:
+        arch: resnext50_32x4d
+        disable_layer_3: true
+        disable_layer_4: true
+    model_path: <path to the model>
+    vocab_path: vocabs/vocab_medium.json
+    ```
+The model can be used either for recognizing rendered formulas and for recognizing scanned formulas (e.g. from a scanner or from a phone camera)
```

Comment:
either ... or ...
both ... and ...

Reply:
Fixed

Reply:
How the user's supposed to choose one of these two options?

Reply:
Reformulated this sentence, it was meant that model is able to recognize both rendered and scanned formulas, and user does not need to make any choice, model supports this functionality by default.